apps/stable_diffusion/scripts/txt2img.py

```diff
@@ -160,19 +167,32 @@ def txt2img_inf(
         seeds.append(img_seed)
         txt2img_obj.log += "\n"
 
-    total_time = time.time() - start_time
-    text_output = f"prompt={args.prompts}"
-    text_output += f"\nnegative prompt={args.negative_prompts}"
-    text_output += f"\nmodel_id={args.hf_model_id}, ckpt_loc={args.ckpt_loc}"
-    text_output += f"\nscheduler={args.scheduler}, device={device}"
-    text_output += (
-        f"\nsteps={steps}, guidance_scale={guidance_scale}, seed={seeds}"
-    )
-    text_output += f"\nsize={height}x{width}, batch_count={batch_count}, batch_size={batch_size}, max_length={args.max_length}"
-    text_output += txt2img_obj.log
-    text_output += f"\nTotal image generation time: {total_time:.4f}sec"
+        total_time = time.time() - start_time
+        text_output = f"prompt={args.prompts}"
+        text_output += f"\nnegative prompt={args.negative_prompts}"
+        text_output += (
+            f"\nmodel_id={args.hf_model_id}, ckpt_loc={args.ckpt_loc}"
+        )
+        text_output += f"\nscheduler={args.scheduler}, device={device}"
+        text_output += (
+            f"\nsteps={steps}, guidance_scale={guidance_scale}, seed={seeds}"
+        )
+        text_output += f"\nsize={height}x{width}, batch_count={batch_count}, batch_size={batch_size}, max_length={args.max_length}"
+        text_output += (
+            f"\n\nTotal image(s) generation time: {total_time:.4f}sec"
+        )
+        print(txt2img_obj.log)
+        txt2img_obj.log = ""
+        if i != batch_count - 1:
+            stable_diffusion_button_label = (
+                f"Processing batch {i+2} / {batch_count}"
+            )
+        else:
+            # This is the last inference: we restore the original button label
+            stable_diffusion_button_label = "Generate Image(s)"
+        yield generated_imgs, text_output, stable_diffusion_button_label
 
```

Comment:
we should only yield generated_imgs here. It's possible to yield subset of total number of outputs expected.

Reply:
Output logs are now stricly identical to the original one, expect I report a progress in the output field when batch is running:
"Processing batch numberx / fromy"